src/main/views/status-description.njk

```diff
@@ -0,0 +1,71 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{{ displayGlossaryAlphabet }}
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ "Event Glossary" | safe }}
+{% endblock %}
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: text,
+    href: '#',
+    attributes: {
+      onClick: "history.back();"
+    }
+  }) }}
+{% endblock %}
+{% block content %}
+  <div id="grid-row" class="govuk-grid-row">
```

Comment:
could this be done in nunjuks format instead? you done seem to be editing any cell contents so HTML might not be essential for this, if we can use nunjuks itll be best to use it where possible

Reply:
nunjuks is not possible in this case because we are using for loop to populate the row. If I injected for loop inside the table template, it broke the page. And on court listing page, we are using simple table not nunjuks.